pandas/tests/frame/methods/test_reindex.py

```diff
@@ -864,3 +864,12 @@ def test_reindex_signature(self):
             "fill_value",
             "tolerance",
         }
+
+    def test_reindex_fill_added_rows(self):
```

Comment:
can you add "multiindex" in the name somehow, and "fill" -> "ffill"

half-sentence after the GH ref below on what the original problem was?

and total nitpick exclusively if you want to do me a kindness: I like the pattern `# GH#23693` for the GH references

Reply:
All done :)